src/components/controller/loader.ts

```diff
@@ -0,0 +1,49 @@
+import { IDataSources } from "../view/sources/sources";
+
+export type Callback<T> = (data: T) => void;
+class Loader {
+    baseLink: string;
+    readonly options: { [key: string]: string };
```

Comment:
Please make the fields of a class private.

Reply:
Done, add access modifiers to the arguments for the constructor.